atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/KvsSweepQueueTest.java

```diff
@@ -86,120 +86,121 @@ public void scrubberCalledEvenIfNothingToSweep() {
 
     @Test
     public void sweepStrategyNothingDoesNotPersistAntything() {
-        sweepQueue.enqueue(writeToDefaultCell(TABLE_NOTHING, TS), TS);
-        sweepQueue.enqueue(writeToDefaultCell(TABLE_NOTHING, TS2), TS2);
-        verify(kvs, times(0)).put(eq(TargetedSweepTableFactory.of().getSweepableCellsTable(null).getTableRef()),
-                anyMap(), anyLong());
-        verify(kvs, times(0)).put(eq(TargetedSweepTableFactory.of().getSweepableTimestampsTable(null).getTableRef()),
-                anyMap(), anyLong());
-        verify(kvs, times(0)).put(eq(TargetedSweepTableFactory.of().getSweepShardProgressTable(null).getTableRef()),
-                anyMap(), anyLong());
+        enqueueWrite(TABLE_NOTHING, TS);
+        enqueueWrite(TABLE_NOTHING, TS2);
+        verify(kvs, times(2)).put(any(TableReference.class), anyMap(), anyLong());
```

Comment:
i.e. just TABLE_NOTHING? You could verify here that (a) we put twice to nothing, and (b) we don't put elsewhere.

Reply:
Added a line in current branch to prevent merge conflicts